server.js

```diff
@@ -34,14 +38,24 @@ function deploy(config, ready) {
     function toPosixPath(url) {
         return path.posix.join(...url.split(path.sep));
     }
-
-    let logger = fs.createWriteStream('log.txt', {
-        flags: 'a'
-    });
-    let writeLine = (line) => logger.write(`\n${line}`);
+    
+    let writeLine = (line) => {
+        if (config.log != "off") {
+            let path = config.log;
+            // ensure .txt file extension is available
+            if (!path.endsWith(".txt")) {
+                path += ".txt";
+            }
```

Comment:
```suggestion
```
this is not needed. If the developer wants to create a file without extension, so be it